src/Upgrade/Application/Strategy/Common/PackagePostUpdateHandler/PropelUpdateHandler.php

```diff
@@ -42,7 +42,9 @@ class PropelUpdateHandler implements HandlerInterface
         'data/cache/propel',
         'src/Orm/Zed/**/Persistence/Base/',
         'src/Orm/Zed/**/Persistence/Map/',
-        'src/Orm/Propel/*',
+        'src/Orm/Propel/Schema/*',
```

Comment:
have you tried `'src/Orm/Propel/**'`

Reply:
We don't need remove everything from propel dir. Should remove specific folders